sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/aio/_client_async.py

```diff
@@ -99,7 +125,27 @@ async def wait_until_done(self, job_id, **kwargs):
         :return: JobStatusDetail
         :rtype: JobStatusDetail
         """
-        pass
+        pipeline_response = await self.get_job_status(
```

Comment:
same comment as the sync implementation PR, let's call the generated code equivalent here